app/styles/local.css

```
@@ -1,593 +1,593 @@
+ * - https://getbootstrap.com/docs/5.0/migration/
+ */
+
+/*  VALIDATION - FORM-GROUPS */
+
+.form-group .form-text {
+  font-size: 16px;
+  display: block;
+  margin-top: 5px;
+  margin-bottom: 10px;
+  color: #737373;
+  font-size: 16px;
```

Biome (2.4.7)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)